content/articles/heroku-connector.md

```diff
@@ -25,31 +25,35 @@ categories:
 
 ## Creating a connection
 
-1. Head to the "Connections" tab for the domain you want to connect to Heroku. From there, click the "Add" button next to the Heroku connector.
-   ![Create a connection to Heroku](/files/heroku-connector-add-connector.png)
+1. Choose the relevant domain from your Domain List.
+2. Click the **Connections** tab on the left side.
+3. Under **Add Connection**, click **Add** in the **Heroku Connector** section.
+    ![screenshot of adding a connector](/files/add-connector.png)
```

Comment:
@dperrymorrow I accidentally used the netlify one here, should be @dperrymorrow this is the wrong screenshot, should be https://drive.google.com/file/d/1NVmoae4G9bXBC4kVQtg8uZqxqq08EXdr/view?usp=sharing